docs/api-reference/1_5/documents/_overview.md

```diff
@@ -0,0 +1,92 @@
+# <strong>API OVERVIEW</strong>
+
+Welcome to the Kubernetes API.  You can use our API to access Kubernetes API endpoints to read and write Kubernetes
+resource objects.
+
+## Resource Categories
+
+This is a highlevel overview of the basic types of resources provide by the Kubernetes API and their primary functions.
+
+**Workloads** *Run your containers*: These are responsible for managing and running your containers on the cluster.
+
+**Discovery & LB Resources** *Access your containers*: These are responsible for stitching your workloads together into an externally accessible Loadbalanced Service.
+
+**Config & Storage Resources** *Inject initialization data into your containers*: These are responsible for injecting data into your applications and persisting data externally to your container.
```

Comment:
**Configuration and Storage** resources are objects you use to inject initialization data into your applications, and to persist data that is external to your container.